### ISSUE TITLE
Create screenshot from video at specified time

### ISSUE THREAD

**Issue by owcz** [toolforge/video2commons#89]
(I thought I had opened a ticket for this previously, so apologies if I missed it.)

Sometimes all Commons really needs from a video is a single screenshot. It would be nice if there was a way to specify [the timestamp ](https://commons.wikimedia.org/wiki/Template:From_YouTube) for the screenshot and take a single jpg/png rather than uploading the whole video.

This said, it would be nice if such a feature could work retroactively on videos previously uploaded to Commons... so not sure how best to handle, though I imagine your suite already has most of the tools implemented

**Comment by zhuyifei1999** [comment]
That would need a quite different workflow, as the uploaded files aren't oggs or webms.

**Comment by VDK** [comment]
Yes, it is outside the scope of this project. It'd be a good proposal for the community wish list.
